newton/_src/viewer/viewer.py

```
@@ -443,16 +465,23 @@ def finalize(self):
             self.scales = wp.array(self.scales, dtype=wp.vec3, device=self.device)
             self.colors = wp.array(self.colors, dtype=wp.vec3, device=self.device)
             self.materials = wp.array(self.materials, dtype=wp.vec4, device=self.device)
+            self.worlds = wp.array(self.worlds, dtype=int, device=self.device)
 
             self.world_xforms = wp.zeros_like(self.xforms)
 
-        def update(self, state):
+        def update(self, state, world_offsets=None):
             from .kernels import update_shape_xforms  # noqa: PLC0415
```

Ruff (0.14.1)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)